engines/account/app/workers/account/signup_worker.rb

```diff
@@ -0,0 +1,13 @@
+module Account
+  class SignupWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low
+
+    def perform(signup_id)
+      signup = Signup.find_by(id: signup_id)
+
+      SignupMailer.confirm_email(signup).deliver
```

Comment:
@rewritten @andresgutgon totally agree with migrate to `4.2`. Right now the project has few dependencies and It'd easier to do now than wait till we have more features. Also I'd rather use `ActionMailer#deliver_later` for this case of asynchronous mails